Bot doesn't play music at all

### Bug Description

It started on 7nd August

### Steps to Reproduce

Launch the bot.
Play something from YouTube.
Bot just joins the channel and nothing.

### Expected Result

Bot should play music.

### Debug Output

[00:22:03] [ERROR] [SignatureCipherManager]: Problematic YouTube player script /s/player/b12cc44b/player_ias.vflset/ru_RU/base.js detected (issue detected with script: no n function match). Dumped to C:\Users\shoke\AppData\Local\Temp\lavaplayer-yt-player-script12383617030419906327.js
[00:24:38] [ERROR] [LocalAudioTrackExecutor]: Error in playback of TVW2KZlxvWI
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
    at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45)
    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121)
    at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: Not success status code: 403
    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:158)
    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:174)
    at java.base/java.io.DataInputStream.readUnsignedByte(Unknown Source)
    at java.base/java.io.DataInputStream.readByte(Unknown Source)
    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:47)
    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:94)
    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:52)
    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:35)
    at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
    at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:148)
    at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:116)
    at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)


### Additional Info

_No response_

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am using an official release from [the releases page](https://github.com/jagrosh/MusicBot/releases)

### JMusicBot Version

0.4.3

same issue but

``` nohup: ignoring input
[15:21:57] [INFO] [JMusicBot]: Loaded config from /www/wwwroot/bot/config.txt
[15:21:57] [INFO] [Settings]: serversettings.json loaded from /www/wwwroot/bot/serversettings.json
[15:21:58] [ERROR] [JMusicBot]: Could not start GUI. If you are running on a server or in a location where you cannot display a window, please run in nogui mode using the -Dnogui=true flag.
[15:21:58] [INFO] [JDA]: Login Successful!
[15:21:59] [INFO] [WebSocketClient]: Connected to WebSocket
[15:22:00] [INFO] [JDA]: Finished Loading!
[13:19:32] [ERROR] [SignatureCipherManager]: Problematic YouTube player script /s/player/20dfca59/player_ias.vflset/en_GB/base.js detected (issue detected with script: no n function match). Dumped to /tmp/lavaplayer-yt-player-script8180808654484691734.js
[14:07:55] [ERROR] [SignatureCipherManager]: Problematic YouTube player script /s/player/d2e656ee/player_ias.vflset/en_GB/base.js detected (issue detected with script: no n function match). Dumped to /tmp/lavaplayer-yt-player-script14719962051434683381.js
[13:00:21] [ERROR] [SignatureCipherManager]: Problematic YouTube player script /s/player/bd3293c9/player_ias.vflset/en_GB/base.js detected (issue detected with script: no n function match). Dumped to /tmp/lavaplayer-yt-player-script18022819279653247172.js
[13:01:12] [INFO] [NativeLibraryLoader]: Native library libmpg123-0: loading with filter com.sedmelluq.discord.lavaplayer.natives.ConnectorNativeLibLoader$$Lambda$355/0x00000008403a7840@34bd8457
[13:01:12] [INFO] [NativeLibraryLoader]: Native library connector: loading with filter null
[13:01:12] [INFO] [NativeLibraryLoader]: Native library connector: successfully loaded.
[08:56:01] [ERROR] [AudioHandler]: Track QvMXKP1awYM has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[08:56:20] [ERROR] [AudioHandler]: Track QvMXKP1awYM has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[08:56:48] [ERROR] [AudioHandler]: Track QvMXKP1awYM has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[13:32:13] [ERROR] [SignatureCipherManager]: Problematic YouTube player script /s/player/b12cc44b/player_ias.vflset/en_GB/base.js detected (issue detected with script: no n function match). Dumped to /tmp/lavaplayer-yt-player-script17315652341206492158.js
[13:32:14] [ERROR] [AudioHandler]: Track QvMXKP1awYM has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[13:39:06] [ERROR] [AudioHandler]: Track lyYrvaXMRHs has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[13:40:27] [ERROR] [AudioHandler]: Track QvMXKP1awYM has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Sign in to confirm you’re not a bot
	at dev.lavalink.youtube.clients.skeleton.Client.getPlayabilityStatus(Client.java:74)
	at dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:98)
	at dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:183)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:109)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:71)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829) ```

--- a/pom.xml
+++ b/pom.xml
@@ -60,7 +60,7 @@
         <dependency>
             <groupId>dev.lavalink.youtube</groupId>
             <artifactId>common</artifactId>
-            <version>1.5.1</version>
+            <version>1.5.2</version>
         </dependency>
 	<dependency>
 	    <groupId>com.github.jagrosh</groupId>

